integration-tests/tests/register-user.js

```diff
@@ -23,21 +27,36 @@ const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
   // We wait here for the cookie message to disappear, it has a fixed duration fading animation of 300ms
   await delay(350);
   await verifyAccessibility(page);
-  await click(page, clickable({ textContains: 'View Workspaces' }));
+
+  // This is the hamburger menu
+  await click(page, clickable({ textContains: 'Toggle main menu' }));
+  // Wait for sidebar to transition in.
+  // Matches transition durations in ModalDrawer.
+  await delay(200);
+
+  await click(page, clickable({ textContains: 'Sign In' }));
+
   await signIntoTerra(page, { token });
   await fillInReplace(page, input({ labelContains: 'First Name' }), 'Integration');
   await fillIn(page, input({ labelContains: 'Last Name' }), 'Test');
   await fillIn(page, input({ labelContains: 'Organization' }), 'Test Organization');
   await fillIn(page, input({ labelContains: 'Department' }), 'Test Department');
   await fillIn(page, input({ labelContains: 'Title' }), 'Test Title');
   await fillInReplace(page, input({ labelContains: 'Contact Email for Notifications' }), 'ltcommanderdata@neighborhood.horse');
+
+  // Accept the Terms of Service
+  await click(page, clickable({ textContains: 'Read Terra Platform Terms of Service here' }));
+  await waitForModal(page);
+  await waitForNoSpinners(page);
+  await click(page, button({ text: 'OK' }));
```

Comment:
Why did you need to create  a new method for clicking OK? It seems unusual because a lot of the tests click buttons.

Reply:
Stemmed from my unfamiliarity with the integration test utils. Removed and replaced with `clickable`.